File: Logitude/AngularModules/AngularModules/Accounting/DataContracts/CashbookChequesCounter.ts
export class CashbookChequesCounter
{
    CashChequesCount: number = 0;
    PostdatedChequesCount: number = 0;

    public get AllChequesCount() : number {
        return this.CashChequesCount + this.PostdatedChequesCount;
    }

}

File: Logitude/Logitude.Accounting.BL/DataContract/CashbookChequesCounter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Logitude.Accounting.BL.DataContract
{
    public class CashbookChequesCounter
    {
        public int CashChequesCount { get; set; }
        public int PostdatedChequesCount { get; set; }
        public int AllChequesCount
        {
            get
            {
                return CashChequesCount + PostdatedChequesCount;
            }
        }
    }
}
